### PR TITLE
Ft/adetayo

### PR DESCRIPTION
**Pull request by adetayo2718**
Screenshot 2024-04-16 215747

Implemented the loader, close button, and border in the 
update user modal.

### CODE REVIEW

File: src/shared/Loader.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { CircularProgress } from "@mui/material";
 
 function Loader() {
   return (
-    <div className="flex justify-center items-center w-full h-screen">
+    <div className="flex justify-center items-center w-full ">
```
Comment (pleasantvik) [inline src/shared/Loader.tsx]: Why remove the h-screen here. 

The h-screen was added to ensure the loader is in the middle of the page. Just call the circular progres in the place you wanna use Loader

File: src/views/admin/UserInfo.tsx [hunk 2]
```diff
@@ -57,9 +58,21 @@ const UserInfo = () => {
...
         <form onSubmit={userForm.handleSubmit}>
+          <div className="grid">
+            <CustomButton
+              onClick={() => {
+                setOpenModal(false);
+              }}
+              variant="contained"
+              className="rounded-full max-w-0 justify-self-end"
+            >
+              &#x2715;
```
Comment (pleasantvik) [inline src/views/admin/UserInfo.tsx:71]: What does the line 72 do

File: src/views/admin/UserInfo.tsx [hunk 3]
```diff
@@ -96,8 +109,12 @@ const UserInfo = () => {
...
 
           <div>
-            <CustomButton variant="contained" className=" mt-2" type="submit">
-              Submit
+            <CustomButton
+              variant="contained"
+              className=" min-w-full mt-2"
+              type="submit"
+            >
+              {isLoading ? <Loader /> : "Submit"}
```
Comment (pleasantvik) [inline src/views/admin/UserInfo.tsx]: Just call the CircularProgress component here instead of the loader

**Review by pleasantvik — CHANGES_REQUESTED** [review]
Check the comment

File: src/shared/Loader.tsx (hunk 1 above)
Comment (adetayo2718) [reply src/shared/Loader.tsx]: Its been fixed. I will raise a new PR

File: src/views/admin/UserInfo.tsx (hunk 2 above)
Comment (adetayo2718) [reply src/views/admin/UserInfo.tsx:71]: That is the character code i used to write the X on the close button

File: src/views/admin/UserInfo.tsx (hunk 3 above)
Comment (adetayo2718) [reply src/views/admin/UserInfo.tsx]: Fixed, raising new PR

**Review by pleasantvik — APPROVED** [review]

**Pull request merged**